src/js/video.js

```diff
@@ -42,6 +43,11 @@ videojs.players = Player.players;
 
 videojs.extends = extendsFn;
 
+videojs.getGlobalOptions = () => options;
```

Comment:
I'll add a note. I decided to use it here specifically because this function becomes so short as a fat arrow function.